### DISCUSSION TITLE
Adding Facebook Messenger to the list of End to End encrypted messengers?

### DISCUSSION THREAD

**Discussion by nasbench** [Digital-Forensics-Discord-Server/LawEnforcementResources#15] [General]
Hello,

Not so long ago Facebook announced that messenger got end-to-end encryption on some of the functionalities. I was wondering if it's worth adding to the list of E2E messengers?

Link: https://messengernews.fb.com/2021/08/13/messenger-updates-end-to-end-encrypted-chats-with-new-features/

**Comment by AndrewRathbun** [comment]
Yeah if that's the nature of the app now then we should add it in the corresponding section. Thanks for bringing this up!
